adafruit_adt7410.py

```diff
@@ -36,10 +36,12 @@
 
 import time
 import struct
+import busio
```

Comment:
For the `busio` import lets put it within a try/except and added the necessary pylint exception for the "useless" typing import. There is an example of how we've done it here: https://github.com/adafruit/Adafruit_CircuitPython_NTP/blob/acc7d3608aa33e4740f9ac28eb4d5230b5eb86d7/adafruit_ntp.py#L25-L30

This will make it so that `busio` ends up not actually getting imported when this runs on a microcontroller which will save some RAM.

Reply:
Good thought, thanks! Change added. I'll go through my other pull requests from this morning where I imported busio and make the same change.